analysis/topEFT/make_jsons.py

```diff
@@ -589,17 +589,17 @@
     "UL18_DY10to50" : {
         "path" : "/DYJetsToLL_M-10to50_TuneCP5_13TeV-madgraphMLM-pythia8/RunIISummer20UL18NanoAODv9-106X_upgrade2018_realistic_v16_L1v1-v1/NANOAODSIM",
         "histAxisName": "DY10to50_centralUL18",
-        "xsecName": "DY10to50",
+        "xsecName": "DYJetsToLL_M_10to50_MLM",
     },
     "UL18_DY50" : {
         "path" : "/DYJetsToLL_M-50_TuneCP5_13TeV-amcatnloFXFX-pythia8/RunIISummer19UL18NanoAODv2-106X_upgrade2018_realistic_v15_L1v1-v1/NANOAODSIM",
         "histAxisName": "DY50_centralUL18",
-        "xsecName": "DY50",
+        "xsecName": "DYJetsToLL_M_50_MLM",
     },
     "UL18_singleTop" : {
         "path" : "/ST_s-channel_4f_leptonDecays_TuneCP5_13TeV-amcatnlo-pythia8/RunIISummer20UL18NanoAODv9-106X_upgrade2018_realistic_v16_L1v1-v1/NANOAODSIM",
         "histAxisName": "singleTop_centralUL18",
-        "xsecName": "singleTop",
+        "xsecName": "ST",
```

Comment:
would you be opposed to changing this (and elsewhere) to "ST_s-channel"?

Reply:
Sounds like a good idea.